internal/eventstore/internal/repository/repository.go

```diff
@@ -0,0 +1,19 @@
+package repository
+
+import (
+	"context"
+
+	"github.com/caos/zitadel/internal/eventstore/models"
+)
+
+type Repository interface {
+	Health(ctx context.Context) error
+
+	// PushEvents adds all events of the given aggregates to the eventstreams of the aggregates.
+	// This call is transaction save. The transaction will be rolled back if one event fails
+	PushAggregates(ctx context.Context, aggregates ...*models.Aggregate) error
+
+	PushEvents(ctx context.Context, aggregates [][]*models.Event) (err error)
```

Comment:
Add comment, and change aggregates to events

Reply:
PushEvents is deleted